scripts/legislation/africa/botswana_english.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+
+function remove_all_text_before_first_header {
+  sed -n '/^\[Date of Commencement: 27th May, 2005\]/,$p' | \
+    sed -n '/^PART I/,$p'
+}
+
+function prefix_article_numbers_with_article_literal {
+  sed -E 's/^([0-9]+\.)/Article \1/'
+}
+
+function append_pipe_to_article_title {
+  sed -E 's/^(Article .*)$/\1|/'
+}
+
+function replace_parentheses_around_article_delimiters_with_square_brackets {
+  sed -E 's/^\(([A-Za-z0-9]+)\)/[\1]/' 
+}
+
+function move_article_titles_above_article_bodies {
+  sed -E 's/^(\([0-9]+\)) ([^|]+)\|/\2\n\1/' 
+}
+
+function remove_all_text_after_last_article {
+  sed -E '/^\(32\) /,${/^\(32\)/!d}' | \
+  sed -E 's/\(32\) Articles deemed.*//'
+}
+
+function amend_errors_in_headers {
+    sed -E 's/PART I Preliminary /PART I - Preliminary /' | \
```

Comment:
This first line should not be indented to the same extent as the continuation lines.